src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Colors color = colorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
```

Comment:
Magic number 100 should be replaced with a named constant to improve code readability and maintainability.